src/Security/Security.php

```diff
@@ -514,15 +514,17 @@ protected function getResponseController($title)
             return $this;
         }
 
-        // Use sitetree pages to render the security page
-        $tmpPage = new SiteTree();
+        // Use the default setting for which Page to use to render the security page
+        $pageClass = (string) $this->stat('template_main');
```

Comment:
Oh, risky. template_main could be a path to a template file (e.g. `/themes/mytheme/templates/Security.ss`.

I think this change is probably safe for now, but we should use a class_exists() and document template_main appropriately.